src/app/collection/shared/included/collection-learning-object-card/collection-learning-object-card.component.scss

```diff
@@ -6,71 +6,159 @@ a {
 a:visited {
     color: $darker-grey;
 }
-.container{
-    width: 20vw;
-    margin-bottom: 50px;
-    display: flex;
-}
-.header{
-    &.module{
-        background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380642.jpg");
+
+.wide{
+    .container{
+        width: 20vw;
+        margin-bottom: 50px;
+        display: flex;
     }
-    &.micromodule{
-        background-image: url('../../../../../assets/images/adi-goldstein-EUsVwEOsblE-unsplash\ 2.png');
+    .obj-info{
+        font-size: $normal;
+        .obj-title{
+            padding-top: 20px;
+            font-size: $normal;
+            font-weight: 700;
+            height: 10vh; 
+        }
     }
-    &.nanomodule{
-        background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380618.jpg");
+    .header{
+        &.module{
+            background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380642.jpg");
+        }
+        &.micromodule{
+            background-image: url('../../../../../assets/images/adi-goldstein-EUsVwEOsblE-unsplash\ 2.png');
+        }
+        &.nanomodule{
+            background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380618.jpg");
+        }
+        &.unit{
+            background-image: url("../../../../../assets/images/michael-dziedzic-nLFqr9Mr9H8-unsplash\ 1.png");
+        }
+        &.course{
+            background-image: url("../../../../../assets/images/pexels-scott-webb-430208.jpg");
+        }
+        display: flex;
+        align-items: center;
+        color: #fff;
+        background-size: cover;
+        background-position: center;
+        min-height: 100px;
+        border-radius: 10px;
+        width: 100%;
```

Comment:
This can get moved out to the top level of the scss so we don't need to duplicate it one lines 107-130